README.md

```diff
@@ -1,10 +1,11 @@
 [![Infinispan](https://design.jboss.org/infinispan/logo/final/PNG/infinispan_logo_rgb_lightbluewhite_darkblue_600px.png)](https://infinispan.org/)
 
-[![Version](https://img.shields.io/maven-central/v/org.infinispan/infinispan-core?logo=apache-maven&style=for-the-badge)](https://search.maven.org/artifact/org.infinispan/infinispan-core)
+[![Version Final](https://img.shields.io/maven-central/v/org.infinispan/infinispan-core?logo=apache-maven&style=for-the-badge&versionPrefix=15.0)](https://search.maven.org/artifact/org.infinispan/infinispan-core)
+[![Version Dev](https://img.shields.io/maven-central/v/org.infinispan/infinispan-core?logo=apache-maven&style=for-the-badge&versionPrefix=15.1)](https://search.maven.org/artifact/org.infinispan/infinispan-core)
```

Comment:
Why do we have the Final and Dev links pointing to the same location? 
Shouldn't be https://search.maven.org/artifact/org.infinispan/infinispan-core/15.0.10.Final/jar and https://search.maven.org/artifact/org.infinispan/infinispan-core/15.1.0.Dev04.Final/jar

Reply:
The prefix brings the latest one of each

Reply:
I know, but both points to `https://search.maven.org/artifact/org.infinispan/infinispan-core`